front/tests/utils/api-setup.ts

```diff
@@ -0,0 +1,74 @@
+import { request } from '@playwright/test';
+
+import type { Project, Study, RollingStock, Infra, Scenario } from 'common/api/osrdEditoastApi';
+
+// API requests
+
+export const getApiContext = async () =>
+  request.newContext({
+    baseURL: 'http://localhost:4000',
+  });
+
+export const getApiRequest = async (
+  url: string,
+  params?: { [key: string]: string | number | boolean }
+) => {
+  const apiContext = await getApiContext();
+  const response = await apiContext.get(url, { params });
+  return response.json();
+};
+
+export const postApiRequest = async <T>(
+  url: string,
+  data?: T,
+  params?: { [key: string]: string | number | boolean }
+) => {
+  const apiContext = await getApiContext();
+  const response = await apiContext.post(url, { data, params });
+
+  return response.json();
+};
+
+export const deleteApiRequest = async (url: string) => {
+  const apiContext = await getApiContext();
+  const response = await apiContext.delete(url);
+  return response;
+};
+
+// API calls for beforeAll setup in tests
+
+export const findOneInResults = <T extends { name: string }>(results: T[], name: string) =>
```

Comment:
No need to export this function
```suggestion
const findOneInResults = <T extends { name: string }>(results: T[], name: string) =>
```